swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -62,3 +62,12 @@ func fsLog(message: String, level: LogLevel = .info) {
 func fsModifyPrivacy(setting: PrivacySetting, of view: UIView) {
     FS.addClass(view, className: setting.rawValue)
 }
+
+func fsIdentify(userInfo: [String: Any]) {
+    let userId = "13ff474bae77 \(arc4random())"
```

Comment:
`ServiceManagers/FullStoryManager` is a shared object, correct?

Why create dummy data in this function rather than in a client of the function?

Reply:
Hmm I'm not sure I understand. What do you mean by `client of the function`?

Reply:
This SO answer is a good one: https://stackoverflow.com/questions/46498394/what-is-a-function-client

> The other functions, classes, definitions, header or source files that use this function.

Reply:
Ah I see but should I make a client function  so purpose is to create the dummy data? I have an idea so when im done it will look like:

```
func fsIdentify(userInfo: [String: Any]) {
    FS.identify(userInfo["id"] as? String ?? "", userVars: userInfo)
}
```

Reply:
You already have a client of this function in `StoreViewController.swift` and you're passing in an empty `userInfo`: 
`fsIdentify(userInfo: [:])`

pass dummy data in here, rather than in the body of `fsIdentify` like you are currently.

Reply:
I had just caught on lol